dev-cpp/yaml-cpp/files/yaml-cpp-0.6.3-fix-overflows.patch

```diff
@@ -0,0 +1,140 @@
+diff --git a/include/yaml-cpp/depthguard.h b/include/yaml-cpp/depthguard.h
```

Comment:
Include CVE name if we have one in the file name and a link to where you found this above the diff line.